com.unity.render-pipelines.universal/Shaders/LitInput.hlsl

```diff
@@ -6,8 +6,13 @@
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/SurfaceInput.hlsl"
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/ParallaxMapping.hlsl"
 
+#if defined(_DETAIL_MULX2) || defined(_DETAIL_SCALED)
+#define _DETAIL
```

Comment:
Maybe better to have the define a bit clearer?
`_USING_DETAIL_MAP` / `_DETAILMAP` /....

Reply:
I agree its clearer. However I think it would be very inconsistent with our other defines:
_OCCLUSIONMAP, _CLEARCOATMAP, _PARALLAXMAP, _METALLICSPECGLOSSMAP

Of course those came from builtin and we only kept them for upgradability reasons and _DETAIL is not part of it.
What if I would change it to _DETAILMAP? I just feel _USING_DETAIL_MAP would be too much out of place

Reply:
Yes, you're right. `_DETAILMAP` sounds good 👍